tox.ini

```diff
@@ -34,7 +34,9 @@ commands= py.test --genscript=pytest1
 [testenv:flakes]
 basepython = python2.7
 deps = flake8
+     restructuredtext_lint
```

Comment:
Decided to include the `rst-lint` check into `flakes` testenv because it has the same objective (static checking) and I felt it was a little overkill to create an entire new testenv just for this check.